mowgli_etl/pipeline/wdc/wdc_transformer.py

```diff
@@ -95,16 +51,19 @@ def transform(
         self.__product_type_classifier = wdc_product_type_classifier
 
         # Parse file
-        with open(wdc_clean_file_path) as data:
+        with open(wdc_jsonl_file_path) as data:
             for row in data:
-                product = next(
+                entry = WdcOffersCorpusEntry.from_json(row)
+                product_type = next(
                     self.__product_type_classifier.classify(
-                        entry=WdcOffersCorpusEntry.from_json(row)
+                        entry=entry
                     )
                 )
-                if product.expected:
+                parsed_dimensions = next(self.__dimension_parser.parse(entry=entry))
+                
+                if product_type.expected:
                     yield KgEdge.with_generated_id(
-                        subject=product.expected.name,
+                        subject=product_type.expected.name,
```

Comment:
This will likely yield duplicate edges? but it's OK for now, since it's not a valid edge anyway.